src/vision_annotation.workflows.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START workflows_vision_annotation]
+main:
+  params: [args]
+  steps:
+    - new_annotation:
+        call: http.post
+        args:
+          url: ${"https://vision.googleapis.com/v1/images:annotate"}
+          auth:
+            type: OAuth2
+          body:
+            requests: ${args.requests}
```

Comment:
I think we have 2 options to make the request body clear to users: 1) add a comment above `main:` with the request body or 2) update this to:
```
              - image:
                  source:
                      gcsImageUri: ${args}
                features:
                  - type: "LABEL_DETECTION"
                    maxResults: 1
```

Reply:
Hi @averikitsch I would like to go with option 1. Since it gives the user more flexibility to use the vision API as required by just changing the request.